airbyte-integrations/connectors/destination-databricks/Dockerfile

```diff
@@ -16,5 +16,5 @@ ENV APPLICATION destination-databricks
 
 COPY --from=build /airbyte /airbyte
 
-LABEL io.airbyte.version=0.3.1
+LABEL io.airbyte.version=0.3.2
```

Comment:
let's bump this to 1.0.0, because this is a major breaking change

Reply:
Updated